src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -27,21 +28,26 @@ import {
   systemErrorComponent,
   unavailableComponent,
 } from './utils';
-import '../../sass/download-1095b.scss';
 
 export const App = ({ displayToggle, toggleLoginModal }) => {
   const [year, updateYear] = useState(0);
   const [formError, updateFormError] = useState({ error: false, type: '' });
   const cspId = useSelector(signInServiceName);
   const [verifyAlertVariant, setverifyAlertVariant] = useState(null);
   const profile = useSelector(state => selectAuthStatus(state));
+  const [hasLoadedMostRecentYear, setHasLoadedMostRecentYear] = useState(false);
   const isAppLoading = useMemo(
     () => {
-      return profile.isLoadingProfile;
+      return (
+        profile.isLoadingProfile ||
+        (displayToggle &&
+          profile.isUserLOA3 &&
+          hasLoadedMostRecentYear === false) ||
+        displayToggle === undefined
```

Comment:
Have you seen the display toggle be undefined? I assume this is some transient state that can occur while loading everything.

Reply:
Right, that's what it will evaluate to until a boolean is returned.